### ISSUE TITLE
Server running on the Docker image is slow

### ISSUE THREAD

**Issue by Akeyo21** [lowerquality/gentle#329]
## Issue
Whenever we try to run forced alignment for audios when running the Docker image locally the response takes really long. For a 14 second audio, it takes roughly 5 minutes to get the response. However, when we run forced alignment on the Mac application for the same audio, it takes less than 30 seconds.

Was this how it was intended to work, have the docker image run slower than the Mac application?

The docker image is run on a Mac M2 Pro, with 32GB RAM on OS Ventura 13.4

## How to reproduce

1. Pull the image by running the command:
    ```
   docker pull lowerquality/gentle:latest
   ```

2. Run the image by running the command: 
   ```
    docker run -P lowerquality/gentle
   ```
3. Test the server by either:
     a.  sending a HTTP request either via Postman or curl. For curl, you can run the command: 

          
            curl -F "audio=@audio.mp3" -F "transcript=@words.txt" "http://localhost:8765/transcriptions?async=false"
          

     b. Running the application locally, by typing this in the browser: `http://localhost:[portnumber]/` and try and align audio

**Comment by niranjanakella** [comment]
Yes @Akeyo21, experienced the same because the docker resources aren't enough to infer this system properly. Hence running the native Mac application they provide yielded good results for me.

**Comment by ctstewart** [comment]
@niranjanakella Unfortunately the native Mac application doesn't seem to work anymore.
